dspace/config/modules/authentication-saml.cfg

```diff
@@ -0,0 +1,41 @@
+#---------------------------------------------------------------#
+#---------------SAML AUTHENTICATION CONFIGURATIONS--------------#
+#---------------------------------------------------------------#
+#           Configuration properties used by the SAML           #
+#           Authentication plugin, when it is enabled.          #
+#---------------------------------------------------------------#
+
+# The ID of the SAML relying party we should use for authentication
+# authentication-saml.relying-party-id = auth0
+
+# The base URL of all SAML relying party endpoints
+authentication-saml.relying-party-url = ${dspace.server.url}/saml2
+
+# The URL of the authenticate endpoint for the SAML relying party
+authentication-saml.authenticate-endpoint = ${authentication-saml.relying-party-url}/authenticate/${authentication-saml.relying-party-id}
+
+# Should we allow new users to be registered automatically?
+authentication-saml.autoregister = true
+
+# The request attribute that contains the ID of the SAML relying party that authenticated the user
+authentication-saml.attribute.relying-party-id = org.dspace.saml.RELYING_PARTY_ID
+
+# The request attribute that contains the user's SAML name ID
+authentication-saml.attribute.name-id = org.dspace.saml.NAME_ID
+
+# The request attribute that contains the user's email
+authentication-saml.attribute.email = org.dspace.saml.EMAIL
+
+# The request attribute that contains the user's first name
+authentication-saml.attribute.first-name = org.dspace.saml.GIVEN_NAME
+
+# The request attribute that contains the user's last name
+authentication-saml.attribute.last-name = org.dspace.saml.SURNAME
+
+# Additional attribute mappings. Multiple attributes may be stored for each user. The left side is
+# the request attribute, and the right side is the ePerson metadata field to map the attribute to.
+# authentication-saml.eperson.metadata = \
+#   org.dspace.saml.PHONE => phone
+
+# If the ePerson metadata field is not found, should it be created automatically?
+authentication-saml.eperson.metadata.autocreate = true
```

Comment:
Similar to above, this should be set to "false" by default.  We should also note in the comment that the metadata fields will be created in the existing "eperson" schema.

Just a general comment.  Any of the configurations in this file that have a defined "default value" in Java code can be commented out by default.   I believe that means that most of these configurations can be commented out by default.  (This would be more similar to other existing `authentication-*.cfg` files, where many of the configs are commented out by default.)

Reply:
Fixed in 481cdd1. I also added defaults for some settings that didn't have them, but should have (in a919719).